test/server/graphql/common/transactions.test.js

```diff
@@ -78,9 +89,35 @@ describe('server/graphql/common/transactions', () => {
       expect(await canRefund(transaction, undefined, hostAccountantReq)).to.be.false;
       expect(await canRefund(transaction, undefined, contributorReq)).to.be.false;
       expect(await canRefund(transaction, undefined, fromCollectiveAccountantReq)).to.be.false;
+      expect(await canRefund(transaction, undefined, hostAdminReq)).to.be.true;
+      expect(await canRefund(transaction, undefined, rootAdminReq)).to.be.true;
+      expect(await canRefund(oldTransaction, undefined, hostAdminReq)).to.be.true;
+      expect(await canRefund(oldTransaction, undefined, rootAdminReq)).to.be.true;
+    });
+
+    it('can refund as admin of the receiving collective only if transaction < 30 days old', async () => {
       expect(await canRefund(transaction, undefined, collectiveAdminReq)).to.be.true;
+      expect(await canRefund(oldTransaction, undefined, collectiveAdminReq)).to.be.false;
```

Comment:
(Not a requested change) Would have been great to test that host admins are not impacted by this